### DISCUSSION TITLE
QR scans a failure adress. Can't process the donation

### DISCUSSION THREAD

**Discussion by MODUNICO** [btcpayserver/btcpayserver#6140] [Technical Support]
When the client scans QR code to donate, the code is like that: bitcoin:BC1QUETXDHWEZ3J6JMJKJ4652NQPJXDLV797ANXAV7?amount=0.00017453

It should be just the adress: BC1QUETXDHWEZ3J6JMJKJ4652NQPJXDLV797ANXAV7

it doesnt work as it should
![platnosc szczegol](https://github.com/user-attachments/assets/a85fbf6f-7d82-440e-9090-0cd043754889)

**Comment by pavlenex** [comment]
Which wallet do you use @MODUNICO, it's not a bug the wallet is very likely not following the bip32 specs? The solution here would be to reach out to wallet provider.

**Reply by MODUNICO** [reply]
Yeah thats true. Trustwallet works well. Metamask and Binance don't. But I guess they are foloowing the bip32
